llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -6113,7 +6113,8 @@ AArch64TTIImpl::getShuffleCost(TTI::ShuffleKind Kind, VectorType *DstTy,
   unsigned Unused;
   if (LT.second.isFixedLengthVector() &&
       LT.second.getVectorNumElements() == Mask.size() &&
-      (Kind == TTI::SK_PermuteTwoSrc || Kind == TTI::SK_PermuteSingleSrc) &&
+      (Kind == TTI::SK_PermuteTwoSrc || Kind == TTI::SK_PermuteSingleSrc ||
+       Kind == TTI::SK_InsertSubvector) &&
```

Comment:
Maybe leave a comment about why `SK_InsertSubvector` is included here?

Reply:
Commit 7 adds such a comment.